### PR TITLE
rg-67: add delete button for resumes

### PR DESCRIPTION
**Pull request by ValentinaTenus**

### CODE REVIEW

**Comment by ValentinaTenus** [comment]
delete-resume-button
delete-resume-modal

File: frontend/src/bundles/resume/components/delete-resume-button/delete-resume-button.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        setIsModalShown(true);
+    }, []);
+
+    const handleCloseModal = useCallback(() => {
+        setIsModalShown(false);
+    }, []);
+
+    return (
+        <div className={styles.delete__button__wrapper}>
+            <button onClick={handleShowModal} className={styles.delete__button}>
```
Comment (Vitaliy-Kalenichenko) [inline frontend/src/bundles/resume/components/delete-resume-button/delete-resume-button.tsx:19]: Don't use default button

File: frontend/src/bundles/resume/components/delete-resume-button/delete-resume-button.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                <svg
```
Comment (Vitaliy-Kalenichenko) [inline frontend/src/bundles/resume/components/delete-resume-button/delete-resume-button.tsx:20]: Move it to a separate file

File: frontend/src/bundles/resume/components/delete-resume-button/delete-resume-button.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        setIsModalShown(false);
+    }, []);
+
+    return (
+        <div className={styles.delete__button__wrapper}>
+            <BaseButton
+                onClick={handleShowModal}
+                className={styles.delete__button}
+            >
+                <img src={deleteIcon} alt="delete icon" />
```
Comment (Vitaliy-Kalenichenko) [inline frontend/src/bundles/resume/components/delete-resume-button/delete-resume-button.tsx:26]: It's not the best way of using icons. There should be something like `Icon` component for it

File: frontend/src/assets/img/delete-icon.svg [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<svg width="24" height="24" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg">
```
Comment (Vitaliy-Kalenichenko) [inline frontend/src/assets/img/delete-icon.svg:1]:
```suggestion
<svg width="100%" height="100%" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg">
```

File: frontend/src/assets/img/delete-icon.svg [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+<path d="M8,19 L16,19 L16,13 L8,13 L8,19 Z M16,7 L20,7 L20,9 L4,9 L4,7 L8,7
+    L8,3 L16,3 L16,7 Z M14,7 L14,5 L10,5 L10,7 L14,7 Z M6,11 L18,11 L18,21
+    L6,21 L6,11 Z" fill='rgb(26, 145, 240)' fill-rule="nonzero">
```
Comment (Vitaliy-Kalenichenko) [inline frontend/src/assets/img/delete-icon.svg:4]:
```suggestion
    L6,21 L6,11 Z" fill='currentColor' fill-rule="nonzero">
```

File: frontend/src/bundles/resume/components/icons/delete-icon.tsx
Comment (Vitaliy-Kalenichenko) [inline frontend/src/bundles/resume/components/icons/delete-icon.tsx:1]: Move svg to file, use it as other images

File: frontend/src/bundles/resume/components/icons/styles.module.scss [hunk 6]
```diff
@@ -0,0 +1,5 @@
+.delete__icon {
+    width: 24px;
+    height: 24px;
+    color: rgb(26 145 240);
```
Comment (Vitaliy-Kalenichenko) [inline frontend/src/bundles/resume/components/icons/styles.module.scss:4]: Check if there is variable for this color, if there isn't, add it

File: frontend/src/bundles/common/components/custom-icon/custom-icon.tsx
Comment (Vitaliy-Kalenichenko) [inline frontend/src/bundles/common/components/custom-icon/custom-icon.tsx:1]: Move it to *.svg file

Comment (ValentinaTenus) [reply frontend/src/bundles/common/components/custom-icon/custom-icon.tsx:1]: I know what you mean but I can't do it changes color(((

**Pull request merged**